.github/scripts/publish_container.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+GIT_VERSION=$(git describe --tags --always --match 'v*')
+XYZ_VERSION=${GIT_VERSION%-*-*}
+XY_VERSION=${XYZ_VERSION%.*}
+X_VERSION=${XY_VERSION%.*}
+
+for version in "latest" ${X_VERSION} ${XY_VERSION} ${XYZ_VERSION} ${GIT_VERSION}; do
```

Comment:
we only want `latest` and `GIT_VERSION` on master

we want the `X` and `XY` and `XYZ` on release